sormas-e2e-tests/src/test/resources/features/sanity/web/Event.feature

```diff
@@ -195,54 +195,6 @@ Feature: Create events
     And I fill all fields for a new case created for event participant
     And I click on save case button
 
-    @issue=SORDEV-5915 @env_main @ignore
-  Scenario: Check all filters are work properly in Event directory
-    Given API: I create a new event
-    Then API: I check that POST call body is "OK"
-    And API: I check that POST call status code is 200
-    When I log in with National User
-    And I click on the Events button from navbar
-    Then I select random Risk level filter among the filter options from API
-    And I fill EVENT ID filter by API
-    And I apply on the APPLY FILTERS button from Event
-    And I check that number of displayed Event results is 1
-    Then I select random Risk level filter among the filter options
-    And I apply on the APPLY FILTERS button from Event
-    And I check that number of displayed Event results is 0
-    And I click on the RESET FILTERS button from Event
-    Then I select random Disease filter among the filter options from API
-    And I fill EVENT ID filter by API
-    And I apply on the APPLY FILTERS button from Event
-    And I check that number of displayed Event results is 1
-    Then I select random Disease filter among the filter options
-    And I apply on the APPLY FILTERS button from Event
-    And I check that number of displayed Event results is 0
-    And I click on the RESET FILTERS button from Event
-    Then I click on Show more filters in Events
-    Then I select Source Type among the filter options from API
-    And I fill EVENT ID filter by API
-    And I apply on the APPLY FILTERS button from Event
-    And I check that number of displayed Event results is 1
-    Then I select random Source Type among the filter options
-    And I apply on the APPLY FILTERS button from Event
-    And I check that number of displayed Event results is 0
-    And I click on the RESET FILTERS button from Event
-    Then I click on Show more filters in Events
-    Then I select Type of Place field among the filter options from API
-    And I fill EVENT ID filter by API
-    And I apply on the APPLY FILTERS button from Event
-    And I check that number of displayed Event results is 1
-    Then I select random Type of Place field among the filter options
-    And I apply on the APPLY FILTERS button from Event
-    And I check that number of displayed Event results is 0
-    And I click on the RESET FILTERS button from Event
-    Then I select Signal filter from quick filter
-    And I select Event filter from quick filter
-    And I select Screening filter from quick filter
-    And I select Cluster filter from quick filter
-    And I select Dropped filter from quick filter
-    And I click on the RESET FILTERS button from Event
-
```

Comment:
why was this test removed?

Reply:
This test is moved to EventFilters.feature

Reply:
but I can't see the change here on this PR